FreeRTOS/FreeRTOS
# Add SMP task delete on target test

Add SMP task delete on target test

Description
-----------
Adding the on target test to verify that
> When a n RTOS object is deleted, the associated resources shall be freed.

Test Steps
-----------
Test result on RP2040
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/task_delete/task_delete.c:252:Test_TaskDelete:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
